content/javascript/concepts/objects/objects.md

```diff
@@ -62,11 +62,11 @@ person.lastName;
 Bracket notation must be used if the property name has numbers, spaces, or special characters:
```

Comment:
```suggestion
Bracket notation must be used if the property name has spaces, special characters, or begins with a number:
```

It seems that numbers can be involved in the property name for dot notation as long as the prop name _does not_ begin with a number. (i.e., `obj.3blindMice` would not be valid whereas `obj.the3musketeers` would be)